### ISSUE TITLE
Sitemap wrong (NSLink)

### ISSUE THREAD

**Issue by BilelJegham** [roadiz/BaseTheme#18]
Sitemap index url with error 404.

This was because the nodeSources `NSLink` is index.

```html
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml">
....
    <url>
<loc>http://localhost:8088/fr/mentions-legales</loc>
<xhtml:link rel="alternate" hreflang="en" href="http://localhost:8088/mentions-legales" />
<xhtml:link rel="alternate" hreflang="fr" href="http://localhost:8088/fr/mentions-legales" />
</url>
</urlset>
```

Should we change the nodeType 'Link' ?

**Comment by ambroisemaupate** [comment]
Hello,  @BilelJegham 

I think we should remove `NSLink` from node-type list in SitemapController.
We have to keep its `reachable` state to be able to display link in navigation.

**Comment by BilelJegham** [comment]
Sorry for the delay, 
Thanks for your answer

Unfortunately your fix, research NodeType `!=` and `Link`. I fix this in pull request #19 .
